google/cloud/forseti/services/client.py

```diff
@@ -94,6 +96,39 @@ def run(self, config_dir):
                              metadata=self.metadata())
 
 
+class NotifierClient(ForsetiClient):
+    """Notifier service allows the client to send violation notifications."""
+
+    def __init__(self, config):
+        super(NotifierClient, self).__init__(config)
+        self.stub = notifier_pb2_grpc.NotifierStub(config['channel'])
+
+    def is_available(self):
+        """Checks if the 'Notifier' service is available by performing a ping.
+        """
+
+        data = binascii.hexlify(os.urandom(16))
+        echo = self.stub.Ping(notifier_pb2.PingRequest(data=data)).data
+        return echo == data
+
+    def run(self, config_file, inventory_index_id):
+        """Runs the notifier.
+
+        Args:
+            config_file (str): Path the to the forseti config file.
+            inventory_index_id (int): Inventory Index Id.
+
+        Returns:
+            Any service response.
+        """
+
+        request = notifier_pb2.RunRequest(
+            config_file=config_file,
```

Comment:
config file location should not be sent to the server. If we are ever going to host the server and the client on different machines, it will be a problem. Maybe send the file content or necessary parameters.

Reply:
Good catch.  But instead of sending the file content, I'm going to keep this config file on the server-side, and reference it via flag on server startups.